jmix-templates/content/flowui/detail/controller.java

```diff
@@ -8,12 +11,50 @@
 <%}%>
 import com.vaadin.flow.router.Route;
 import io.jmix.flowui.view.*;
+<%if (useDataRepositories){%>
+import io.jmix.core.LoadContext;
+import io.jmix.core.SaveContext;
+import org.springframework.beans.factory.annotation.Autowired;
 
+import java.util.Set;
+
+import static io.jmix.core.repository.JmixDataRepositoryUtils.*;
+<%}%>
 <%if (classComment) {%>
 ${classComment}
 <%}%>@Route(value = "${detailRoute}/:${detailRouteParam}", layout = <%if (!api.jmixProjectModule.isApplication() || routeLayout == null) {%> DefaultMainViewParent.class <%} else {%>${routeLayout.getControllerClassName()}.class<%}%>)
 @ViewController("${detailId}")
 @ViewDescriptor("${detailDescriptorName}.xml")
 @EditedEntityContainer("${dcId}")
-public class ${detailControllerName} extends StandardDetailView<${entity.className}> {
+public class ${detailControllerName} extends StandardDetailView<${entity.className}> {<%if (useDataRepositories){%>
+
+    @Autowired
+    private ${repository.getQualifiedName()} repository;
+
+
+    @Install(target = Target.DATA_CONTEXT)
+    private Set<Object> saveDelegate(SaveContext saveContext) {
```

Comment:
I get empty line in the beginning of the generated method (before `return Set.of...`):

```java
    @Install(target = Target.DATA_CONTEXT)
    private Set<Object> saveDelegate(SaveContext saveContext) {

        return Set.of(repository.save(getEditedEntity()));
    }
```